clippy_lints/src/operators/numeric_arithmetic.rs

```diff
@@ -47,6 +51,9 @@ impl Context {
 
         let (l_ty, r_ty) = (cx.typeck_results().expr_ty(l), cx.typeck_results().expr_ty(r));
         if l_ty.peel_refs().is_integral() && r_ty.peel_refs().is_integral() {
+            if Self::skip_expr_involving_integers(cx, expr) {
```

Comment:
`skip_expr_involving_integers` includes only `is_from_proc_macro`, so it would be better to just use `is_from_proc_macro` here.
```suggestion
            if is_from_proc_macro(cx, expr) {
```